config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

Comment:
This shouldn't go in, calling `make reset` will remove it.

Reply:
Sure, will do in the next revision.

There are also some failing checks, I will also fix them in the next revision.